pkg/genericapiserver/genericapiserver.go

```diff
@@ -593,7 +593,7 @@ func verifySecurePort(options *options.ServerRunOptions) {
 	}
 }
 
-// TODO: Allow 0 to turn off insecure port.
+// TODO: Not allow 0 to turn off insecure port, it's different from SecurePort.
```

Comment:
The TODO as it stands is correct, IMO it should be possible to turn off the insecure port, but isn't right now.